packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -10,16 +10,33 @@ import 'package:flutter/painting.dart';
 import 'package:vector_math/vector_math_64.dart';
 
 import 'debug.dart';
+import 'node.dart';
 
 /// A composited layer.
 ///
 /// During painting, the render tree generates a tree of composited layers that
 /// are uploaded into the engine and displayed by the compositor. This class is
 /// the base class for all composited layers.
-abstract class Layer extends Object with TreeDiagnosticsMixin {
-  /// This layer's parent in the layer tree
-  ContainerLayer get parent => _parent;
-  ContainerLayer _parent;
+///
+/// Most layers can have their properties mutated. The layer tree as a whole can
+/// be reshaped also, with children being moved around the tree. The scene must
```

Comment:
Yeah. Replaced with: `Most layers can have their properties mutated, and layers can be moved to different parents.`